src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -828,9 +854,13 @@ class TriggerView extends Component<TriggerViewProps, TriggerViewState> {
                         () => {
                             preventBodyScroll(false)
                             this.getWorkflowStatus()
+                            if(this.props.isJobView) {
```

Comment:
Why is this required?

Reply:
Because this is called to fetch data , which is only needed in jobs.